tests/test_scheduler.py

```diff
@@ -631,28 +699,30 @@ def test_inference_plms_no_past_residuals(self):
             scheduler.step_plms(self.dummy_sample, 1, self.dummy_sample).prev_sample
 
     def test_full_loop_no_noise(self):
```

Comment:
@patri in https://github.com/huggingface/diffusers/commit/394243ce98c79e273732f8e09b2e5c50ee9a9bd7 you change PNDM step to accept the timestep value rather than iteration number, but I think the tests were kept as is. I changed the full loop to use actual timesteps.